packages/frontend-2/components/settings/Dialog.vue

```diff
@@ -19,12 +19,12 @@
               <UserIcon class="h-5 w-5" />
             </template>
             <LayoutSidebarMenuGroupItem
-              v-for="(sidebarMenuItem, key) in menuItemConfig.user"
+              v-for="(sidebarMenuItem, key) in userMenuItems"
               :key="key"
               :label="sidebarMenuItem.title"
-              :class="{
-                'bg-highlight-2 hover:!bg-highlight-2': targetMenuItem === key
-              }"
+              :class="[
+                targetMenuItem === key && 'bg-highlight-2 hover:!bg-highlight-2'
```

Comment:
I don't think this is simpler than the check was this before. This relies on you understanding that the last truthy value of an AND check in JS is the one that gets returned - seems overly clever and I'd avoid it. Previous syntax was more self explanatory: `"expected classes": whetherTheyShouldBeOn"`

Reply:
Personally I find this syntax more readable, but no strong preference, happy to change it back :)